src/main/java/entity/Saucer.java

```diff
@@ -160,6 +161,11 @@ public final void update(final List<String> input) {
 		wrapAround();
 		changeDirection();
 		shoot();
+		if (isSmall()) {
+			getThisGame().getAudio().play(Audio.UFOSMALL);
+		} else {
+			getThisGame().getAudio().play(Audio.UFOBIG);
+		}
```

Comment:
Ik weet niet waarom, maar de ufo geluiden zijn erg luid en ze zijn niet gelijk aan wat ze in de originele versie waren.